zabbix/files/default/etc/zabbix/zabbix_agentd.conf.jinja

```diff
@@ -190,7 +190,9 @@ ServerActive={{ settings.get('serveractive', defaults.serveractive) }}
 # Default:
 # Hostname=
 
+{% if settings.get('hostnameitem', defaults.get('hostnameitem', False)) == False -%}
 Hostname={{ settings.get('hostname', defaults.get('hostname', salt['grains.get']('id'))) }}
+{% endif %}
 
 ### Option: HostnameItem
```

Comment:
Thanks for PR, overall looks good for me, but I suggest this variant:

```suggestion
# Hostname=
{% if not settings.get('hostnameitem', defaults.get('hostnameitem', False)) %}
Hostname={{ settings.get('hostname', defaults.get('hostname', salt['grains.get']('id'))) }}
{% endif %}
### Option: HostnameItem
```

it's more "pythonic" and also it won't add extra lines to Zabbix Agent config file of every single user of this formula who currently don't use `HostnameItem`.
I tested it - works fine. Could you, please, test it too?

Also you need to update your commit message a little bit, please, check out [this docs](https://github.com/saltstack-formulas/.github/blob/master/CONTRIBUTING.rst#relationship-between-commit-type-and-version-bump) type must be `fix` not `bugfix` otherwise it won't pass commit message linting. I suggest to use `fix(agent):` to get nice description in release info.

Reply:
Thanks for the feedback. I have changed it to your suggestion and pushed an update.